dsa-problems/leetcode-problems/0900-0999.md

```diff
@@ -537,7 +537,7 @@ export const problems = [
         problemName: "987. Vertical Order Traversal of a Binary Tree",
         difficulty: "Hard",
         leetCodeLink: "https://leetcode.com/problems/vertical-order-traversal-of-a-binary-tree",
-        solutionLink: "#"
+        solutionLink: "/dsa-solutions/lc-solutions/0900-0999/0987-vertical-order-traversal-of-a-binary-tree"
```

Comment:
replace `0987-vertical-order-traversal-of-a-binary-tree`  to `vertical-order-traversal-binary-tree`